Explorer/Assets/Scripts/ECS/SceneLifeCycle/SceneFacade/ISceneFacadeExtensions.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections.Generic;
+using Cysharp.Threading.Tasks;
+using ECS.SceneLifeCycle;
+using UnityEngine;
+
+namespace SceneRunner.Scene
+{
+    public static class ISceneFacadeExtensions
+    {
+        public static void DisposeSceneFacadeAndRemoveFromCache(this ISceneFacade sceneFacade, IScenesCache scenesCache,
+            IReadOnlyList<Vector2Int> parcels)
+        {
+            //TODO: Misha, DisposeAsync is callen in many places. Should all of them add the parcels back to the cache?
+            sceneFacade.DisposeAsync().Forget();
```

Comment:
The only other place is `RealmController`, right? We don't need to individually clean the cache in that case: it's handed by `scenesCache.Clear`:
```
private void FindLoadedScenes(World world)
        {
            allScenes.Clear();
            allScenes.AddRange(scenesCache.Scenes);

            // Dispose all scenes
            scenesCache.Clear();

            // find all loaded scenes
            world.Query(in SCENES, (ref ISceneFacade scene) => allScenes.Add(scene));
        }
```

Reply:
Yes, this flow is for testing only so we don't need any kind of cache there